src/utils/options.ts

```diff
@@ -105,6 +105,11 @@ const recoveryCodeFile = new commander.Option(
   'Path to Recovery Code',
 );
 
+const divertSensitiveFile = new commander.Option(
+  '-dsf, --divert-sensitive-file <path>',
+  'Path to divert sensitive information',
+);
+
```

Comment:
This is too generic in name and description. It's only used for outputting the `RecoveryCode` to a file. So it should be named as such. Also the description needs to explain that it is only used if a `RecoveryCode` is generated.

Reply:
Only concern I have here is there already exists an option named `recoveryCodeFile` so I can't really reuse that name.